### DISCUSSION TITLE
set value Field in Dialog when lookupeditor dropdown changes

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5422] [Help]
![image](https://user-images.githubusercontent.com/24252615/106467996-6f572000-64ae-11eb-8773-2964acb8c6ab.png)

When the turId value in satısrow changes, I want the profit value of that type in turrow to be displayed in the profit field. I have to write my sql query and if so how can I do it
For example, I want the profit value of the tour that has ID 1 when the turID is 1
or

![image](https://user-images.githubusercontent.com/24252615/106469401-4768bc00-64b0-11eb-9dcd-76877f4b3149.png)

Is it possible to do something like this

**Comment by edwardch** [comment]
Try something like this in your constructor in abcGrid.ts
```ts
this.form.TurId.changeSelect2(() => {
    YourModule.xyzService.Retrieve({
        EntityId: Q.toId(this.form.TurId.value)
    }, response => {
        this.form.StatisKar.value = response.Entity.Kar.value;
    });
});
```
